noir_stdlib/src/lib.nr

```diff
@@ -84,9 +84,9 @@ fn verify_proof_internal<let N: u32, let M: u32, let K: u32>(
 #[builtin(assert_constant)]
 pub fn assert_constant<T>(x: T) {}
 
-// Asserts that the given value is both true and known at compile-time
+// Asserts that the given value is both true and known at compile-time.
```

Comment:
Please update the docs with the new behavior

Reply:
You mean, talking about message? What should I say about it?

Reply:
We can say something like "The message can be a string, a format string, or any value, as long as it is known at compile-time". What do you think?

Reply:
> You mean, talking about message?

Yes, and also [here](https://github.com/noir-lang/noir/blob/b8d04d6f1c5dd0bd304df970bf433f5fa60d826a/docs/docs/noir/concepts/assert.md?plain=1#L48), e.g. as mentioned for assert [here](https://github.com/noir-lang/noir/blob/b8d04d6f1c5dd0bd304df970bf433f5fa60d826a/docs/docs/noir/concepts/assert.md?plain=1#L31C1-L32C1).

> We can say something like "The message can be a string, a format string, or any value, as long as it is known at compile-time". What do you think?

Sounds great

Reply:
Done!